homeassistant/components/cover/zwave.py

```diff
@@ -91,18 +91,26 @@ def current_cover_position(self):
 
     def open_cover(self, **kwargs):
         """Move the roller shutter up."""
-        self._network.manager.pressButton(self._open_id)
+        if self._workaround == workaround.WORKAROUND_NO_UPDOWN:
+            self.node.set_dimmer(self.values.primary.value_id, 100)
```

Comment:
Maybe instead of using a workaround we should just check if `self._open_id` is `None`?